styles/variables-silicon.less

```diff
@@ -176,39 +176,51 @@
 
 // Input
 // ---------------------------------------
-@agate-input-border-width: 3px;
+@agate-input-decorator-border-bottom-width: 3px;
 @agate-input-decorator-border-width: 0;
 @agate-input-decorator-focus-border-width: 0;
+@agate-input-decorator-has-value-opacity: 0.7;
 @agate-input-decorator-height: 60px;
+@agate-input-decorator-opacity: 0.4;
+@agate-input-disabled-opacity: 0.3;
 @agate-input-focused-opacity: 1;
-@agate-input-font-size: 28px;
-@agate-input-height: 33px;
-@agate-input-opacity: 0.5;
-@agate-input-icon-display: flex;
-@agate-input-icon-align: center;
-@agate-input-padding-top-bottom: 21px;
-@agate-input-padding-right: 33px;
-@agate-input-padding-left: 0;
-@agate-input-icon-after-padding-top-bottom: 21px;
-@agate-input-icon-after-padding-right: 21px;
+@agate-input-font-size: 30px;
+@agate-input-has-value-opacity: 0.6;
+@agate-input-hasiconbefore-padding-left: 6px;
+@agate-input-height: 54px;
 @agate-input-icon-after-padding-left: 9px;
-@agate-input-icon-before-padding-top-bottom: 21px;
-@agate-input-icon-before-padding-right: 9px;
+@agate-input-icon-after-padding-right: 21px;
+@agate-input-icon-after-padding-top-bottom: 27px;
+@agate-input-icon-align: center;
 @agate-input-icon-before-padding-left: 21px;
-@agate-input-text-align: left;
+@agate-input-icon-before-padding-right: 0;
+@agate-input-icon-before-padding-top-bottom: 21px;
+@agate-input-icon-display: flex;
```

Comment:
do we need this? There may be something I missed.

Reply:
I removed the flex as there was no reason for it.